Userland/DynamicLoader/CMakeLists.txt

```diff
@@ -30,6 +30,11 @@ set(SOURCES ${LOADER_SOURCES} ${AK_SOURCES} ${ELF_SOURCES} ${LIBC_SOURCES1} ${LI
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -nostdlib -pie -fpic -DNO_TLS")
 
+if ("${SERENITY_ARCH}" STREQUAL "aarch64")
+    # On aarch64 the stack protector will be accessed before the Loader can relocate itself..
```

Comment:
nits: s/will/would/, and one period at the end is enough.

Also the first `,` in the commit description should be omitted I think.

Reply:
Fixed!